src/style.css

```diff
@@ -1,3 +1,86 @@
-body {
+* {
   margin: 0;
+  padding: 0;
+}
+
+:root {
+  --blue: #00ACDC;
+}
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
+.header {
+  box-sizing: border-box;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  text-align: center;
+  background: #FFF;
+  width: 100%;
+  box-shadow: 0 2px 4px 0 rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  position: relative;
+  box-sizing: border-box;
+  left: 50px;
+  width: 40px;
+  height: 40px;
+}
+
+
+.nav {
+  position: relative;
+  right: 50px;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  position: relative;
+  margin: 0 10px;
```

Comment:
you should use `:not(:last-child)` or sth like this to work with outer item indents to avoid margin collapse